dev/conductor/core/lib/src/next.dart

```diff
@@ -264,21 +264,33 @@ class NextContext extends Context {
       case pb.ReleasePhase.PUBLISH_VERSION:
         stdio.printStatus('Please ensure that you have merged your framework PR and that');
         stdio.printStatus('post-submit CI has finished successfully.\n');
-        final Remote upstream = Remote(
+        final Remote frameworkUpstream = Remote(
             name: RemoteName.upstream,
             url: state.framework.upstream.url,
         );
         final FrameworkRepository framework = FrameworkRepository(
             checkouts,
             // We explicitly want to check out the merged version from upstream
-            initialRef: '${upstream.name}/${state.framework.candidateBranch}',
-            upstreamRemote: upstream,
+            initialRef: '${frameworkUpstream.name}/${state.framework.candidateBranch}',
+            upstreamRemote: frameworkUpstream,
             previousCheckoutLocation: state.framework.checkoutPath,
         );
-        final String headRevision = await framework.reverseParse('HEAD');
+        final String frameworkHead = await framework.reverseParse('HEAD');
+        final Remote engineUpstream = Remote(
+            name: RemoteName.upstream,
+            url: state.engine.upstream.url,
+        );
+        final FrameworkRepository engine = FrameworkRepository(
```

Comment:
```suggestion
        final FrameworkRepository engine = EngineRepository(
```

Reply:
and then you'll have to move the `tag()` method from `FrameworkRepository` to `Repository` (this should be safe)

Reply:
Thanks, done!